web/writing.html

```diff
@@ -350,6 +350,17 @@ <h5 class="portlet__head">Webhook</h5>
                 </div>
               </div>
 
+              <div class="form-group row">
+                <label for="WebhookRequestHeaders" class="col-sm-2 col-form-label">RequestHeaders</label>
+                <div class="col-sm-10">
+                  <textarea class="form-control form" id="WebhookRequestHeaders" name="WebhookRequestHeaders" rows="1"
```

Comment:
```suggestion
                  <input class="form-control form" id="WebhookRequestHeaders" name="WebhookRequestHeaders" value=""
```

@gutenye 我猜 @jeessy2 的意思是用 `input` 实现输入框默认1行（和 `URL` 一样）。